lte/gateway/c/oai/common/common_defs.h

```diff
@@ -100,6 +100,16 @@ typedef enum {
   TLV_FATAL_ERROR = TLV_VALUE_DOESNT_MATCH
 
 } error_code_e;
+
+/* This enum should match with the ModeMapItem_FederatedMode enum
+ * defined in mconfigs.pb.h
```

Comment:
As `mconfigs.pb.h` is generated from `mconfigs.proto`, better to mention that as reference in the code